components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -211,6 +221,14 @@ func Run(options ...RunOption) {
 	symlinkBinaries(cfg)
 
 	configureGit(cfg)
+	go func() {
```

Comment:
@filiptronicek What about moving the `go fun()...` into `configureGit()´? Feels that would match the pattern we have going here, and reduces the noise on the outside. :+1:

Reply:
Sounds good, will move it to reduce the LOC we have in the main supervisor func. The reason I put it here at first is to not have to pass the cstate around.

Reply:
Fixed in [`0712aea` (#20548)](https://github.com/gitpod-io/gitpod/pull/20548/commits/0712aeac882f4e208925aa9589944c67f8959834)